.travis.yml

```diff
@@ -3,7 +3,7 @@ language: generic
 env:
   global:
     # same as netlify.toml
-    - HUGO_VERSION=0.67.1
+    - HUGO_VERSION=0.75.1
```

Comment:
The version will be updated in another PR (https://github.com/letsencrypt/website/pull/1172)

Could you revert that change?

```suggestion
    - HUGO_VERSION=0.67.1
```

(also, HUGO_SHA256 was not updated, it's the reason the tests fails, but it's not relevant anymore)

Reply:
(that change was overwritten by the merge, nothing to do anymore)